contracts/wallet.sol

```diff
@@ -33,12 +33,53 @@ import "./externals/ERC165.sol";
 import "./externals/ECDSA.sol";
 
 
-/// @title ControllableOwnable combines Controllable and Ownable
-/// @dev providing an additional modifier to check if Owner or Controller
-contract ControllableOwnable is Controllable, Ownable {
-    /// @dev Check if the sender is the Owner or one of the Controllers
-    modifier onlyOwnerOrController() {
-        require (_isOwner(msg.sender) || _isController(msg.sender), "only owner||controller");
+contract NonCustodial2FA is Controllable, Ownable {
+
+    event SetOptOption(address _sender, bool _optOption);
+    event SetPersonal2FA(address _sender, address _p2FA);
+
+    bool public optOut;
```

Comment:
same here - just optOut doesn't explain what opt out is for

Reply:
Check new version :)